src/map/ai/ai_mob_dummy.cpp

```diff
@@ -894,7 +894,7 @@ void CAIMobDummy::ActionAbilityStart()
 		Action.speceffect = SPECEFFECT_HIT;
 		Action.animation  = 0;
 		Action.param	  = m_PMobSkill->getMsgForAction();
-        Action.messageID = m_PMobSkill->getMsg() == 0 ? 0 : 43; //readies message
+        Action.messageID = m_PMobSkill->getMsg() == 0 ? 0 : SKILLTYPE::SKILL_BLU; //readies message (43)

```

Comment:
i think you meant something else here? skill types aren't related in any way to message IDs